src/core/RenderTarget.js

```diff
@@ -38,35 +39,20 @@ class RenderTarget extends EventDispatcher {
 			resolveStencilBuffer: true,
 			depthTexture: null,
 			samples: 0,
-			count: 1
-		}, options );
-
-		const texture = new Texture( image, options.mapping, options.wrapS, options.wrapT, options.magFilter, options.minFilter, options.format, options.type, options.anisotropy, options.colorSpace );
-
-		texture.flipY = false;
-		texture.generateMipmaps = options.generateMipmaps;
-		texture.internalFormat = options.internalFormat;
+			flipY: false,
+			...options,
+		} );
 
 		this.textures = [];
 
-		const count = options.count;
+		const count = options.count !== undefined ? options.count : 1;
 		for ( let i = 0; i < count; i ++ ) {
 
 			this.textures[ i ] = texture.clone();
 			this.textures[ i ].isRenderTargetTexture = true;
 
 		}
 
-		this.depthBuffer = options.depthBuffer;
```

Comment:
I'm afraid it is not correct to add non-texture related properties into the `setValues()` parameter. `options `  should only hold values which are properties of textures.

Reply:
The examples print now a lot of warnings. E.g. `misc_boxselection`:

<img width="543" alt="image" src="https://github.com/mrdoob/three.js/assets/12612165/0715a7b4-7ec2-4572-8035-4bfca3093423">

Reply:
This PR doesn't generally discern options belonging to the render target and its underlying texture.

Need to do that all over.

Reply:
Okay, I'll convert the PR to a Draft-PR in the meanwhile. Convert it back when it is ready for a review 👍